windows/src/developer/TIKE/compile/MergeKeyboardInfo.pas

```diff
@@ -418,6 +424,114 @@ procedure TMergeKeyboardInfo.CheckOrAddID;
     json.AddPair('id', FID);
 end;
 
+procedure TMergeKeyboardInfo.AddSubtagNames(id: String; var o: TJSONObject);
+var
+  displayName, languageName, scriptName, regionName: String;
+  v: TJSONValue;
+begin
+  if id = '' then
+    Exit;
+  with TBCP47Tag.Create(id) do
```

Comment:
I have started deprecating use of `with` because it leaves us with ambiguous scope. Instead, use a variable (`bcp47 := TBCP47Tag.Create(id); try ... finally bcp47.Free; end;`